osama-breakout-room/zeena_kareem/style.css

```diff
@@ -0,0 +1,123 @@
+*{
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+  
+}
+.container{
+    background: rgb(231, 222, 222);
+    box-shadow:0px 0px 5px 0px gray ;
+}
+.container_image{
+    margin-left:25%;
+    margin-right:25%;
+    
+}
+ .container_image img{
+    width:50px;
+    height: 50px;
+   
+}
+.container_image a{
+    margin-top: 8px;
+    margin-left:10px;
+    padding:5px;
+}
+
+.container_image a,#info a{
+    
+    float:right;
+    text-decoration: none;
+    font-size: 12px;
+    border: 1px solid #1da1f2;
+    color: #1da1f2;
+    background:white;
+    text-align: center;
+    border-radius: 8px;
+}
+
+#a,#info>#f{
```

Comment:
I would recommend not using (>) a lot in your CSS selectors it makes your CSS file hard to read